css/style.css

```diff
@@ -332,6 +333,389 @@ body {
+  height: 259px;
+  margin-right: 22px;
+  margin-top: 70px;
+}
+
+.video__header {
+  font-size: 22px;
+  line-height: 27px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #fff;
+  margin-bottom: 15px;
+}
+
+.video__text {
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 100;
+  color: #fff;
+  margin-bottom: 17px;
+}
+
+.video__link {
+  display: inline-block;
+  width: 140px;
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 11px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #fff;
+  position: relative;
+}
+
+.video__link::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 93px;
+  bottom: 2px;
+  background: url('../icons/arrow.png');
+  width: 45px;
+  height: 8px;
+}
+
+.video__link:hover {
+  color: #b5b5b5;
+}
+
+.video__link:active {
+  color: #fff;
+  -webkit-text-stroke: 1px rgba(0, 0, 0, .5);
+  -webkit-text-fill-color: rgba(255, 255, 255, .4);
+}
+
+.video__link:hover::after {
+  opacity: .4;
+}
+
+/* Третий блок */
+
+.stories__card-wrapper {
+  margin-bottom: 78px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stories__header {
+  font-size: 22px;
+  line-height: 21px;
+  text-transform: uppercase;
+  font-weight: 700;
+  color: #000;
+  text-align: center;
+  margin-bottom: 44px;
+}
+
+.stories-card {
+  max-width: 307px;
+}
+
+.stories-card__img {
+  margin-bottom: 29px;
+}
+
+.stories-card__img:hover {
+  opacity: .4;
+  cursor: pointer;
+}
+
+.stories-card__img_1024,
+.stories-card__img_768 {
+  display: none;
+}
+
+.stories-card__header {
+  font-size: 18px;
+  line-height: 17px;
+  font-weight: 700;
+  color: #000;
+  margin-bottom: 8px;
+}
+
+.stories-card__text {
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 400;
+  color: #000;
+  margin-bottom: 18px;
+}
+
+.stories-card__footer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stories-card__link {
+  display: inline-block;
+  width: 125px;
+  position: relative;
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 11px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.stories-card__link::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 93px;
+  bottom: 2px;
+  background: url('../icons/arrow-2.png');
+  width: 31px;
+  height: 8px;
+}
+
+.stories-card__link:hover {
+  color: #b5b5b5;
+}
+
+.stories-card__link:active {
+  color: #fff;
+  -webkit-text-stroke: 1px rgba(0, 0, 0, .5);
+  -webkit-text-fill-color: rgba(0, 0, 0, .1);
+}
+
+.stories-card__link:hover::after {
+  opacity: .5;
+}
+
+.stories-card__date {
+  font-size: 12px;
+  line-height: 11px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #000;
+  opacity: .3;
+}
+
+.stories-card__date:hover {
+  color: #190000;
+  opacity: .6;
+  cursor: pointer;
+}
+
+.stories__subheader {
+  font-size: 18px;
+  line-height: 17px;
+  font-weight: 700;
+  text-align: right;
+  text-transform: uppercase;
+  color: #000000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5612618)